notes/style-guide.md

```diff
@@ -4,31 +4,44 @@ Style guide for the standard library
 This is very much a work-in-progress and is not exhaustive.
 
 ## File structure
+Agda programs are structured in modules. The first module in each file of the library is the top-level module whose name always matches the filename.
 
-#### Module imports
+#### Module import guidelines
```

Comment:
I don't think we need to add "guidelines", the whole file contains guidelines after all. Anyway they're hopefully more than guidelines, they're rules!

Reply:
Yes, makes sense.